tests/kuttl/tests/dataplane-deploy-multiple-secrets/02-assert.yaml

```diff
@@ -0,0 +1,221 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cert-generic-service1-edpm-compute-0
+  annotations:
+    cert-manager.io/certificate-name: generic-service1-edpm-compute-0
+    cert-manager.io/issuer-group: cert-manager.io
+    cert-manager.io/issuer-kind: Issuer
+    cert-manager.io/issuer-name: rootca-internal
+  labels:
+    hostname: edpm-compute-0
+    osdp-service: generic-service1
+    osdpns: openstack-edpm-tls
+type: kubernetes.io/tls
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cert-generic-service1-edpm-compute-1
+  annotations:
+    cert-manager.io/certificate-name: generic-service1-edpm-compute-1
+    cert-manager.io/issuer-group: cert-manager.io
+    cert-manager.io/issuer-kind: Issuer
+    cert-manager.io/issuer-name: rootca-internal
+  labels:
+    hostname: edpm-compute-1
+    osdp-service: generic-service1
+    osdpns: openstack-edpm-tls
+type: kubernetes.io/tls
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cert-generic-service1-edpm-compute-2
+  annotations:
+    cert-manager.io/certificate-name: generic-service1-edpm-compute-2
+    cert-manager.io/issuer-group: cert-manager.io
+    cert-manager.io/issuer-kind: Issuer
+    cert-manager.io/issuer-name: rootca-internal
+  labels:
+    hostname: edpm-compute-2
+    osdp-service: generic-service1
+    osdpns: openstack-edpm-tls
+type: kubernetes.io/tls
+---
+# validate the alt-names - which is a list with elements that can be in any order
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      template='{{index .metadata.annotations "cert-manager.io/alt-names" }}'
+      names=$(oc get secret cert-generic-service1-edpm-compute-0 -n openstack -o go-template="$template")
+      echo $names > test123.data
+      regex="(?=.*(edpm-compute-0\.internalapi\.example\.com))(?=.*(edpm-compute-0\.storage\.example\.com))(?=.*(edpm-compute-0\.tenant\.example\.com))(?=.*(edpm-compute-0\.ctlplane\.example\.com))"
+      matches=$(grep -P "$regex" test123.data)
+      rm test123.data
+      if [ -z "$matches" ]; then
+         echo "bad match: $names"
+         exit 1
+      else
+         exit 0
+      fi
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openstack-edpm-tls-generic-service1-certs-0
+  labels:
+    numberOfSecrets: "3"
+    secretNumber: "0"
+  ownerReferences:
+  - apiVersion: dataplane.openstack.org/v1beta1
+    kind: OpenStackDataPlaneNodeSet
+    name: openstack-edpm-tls
+type: Opaque
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openstack-edpm-tls-generic-service1-certs-1
```

Comment:
here we are asserting the existence of the three secrets to hold the certs
openstack-edpm-tls-generic-service1-certs-1, openstack-edpm-tls-generic-service1-certs-2, and openstack-edpm-tls-generic-service1-certs-0